windows/windows_taskbar.cc

```diff
@@ -199,3 +199,23 @@ bool WindowsTaskbar::ResetWindowTitle() {
   }
   return true;
 }
+
+bool WindowsTaskbar::IsTaskbarVisible() {
+  HWND hTaskbarWnd;
```

Comment:
Please follow [Google Style Guide for C++](https://google.github.io/styleguide/cppguide.html) (mainly naming of identifiers is concerned here, snake case for local identifiers).

Format the source files with clang-format:
https://github.com/alexmercerind/windows_taskbar/blob/b3273d58aab9d71f61b51a84d5c6fb6cf0821529/.clang-format#L1-L3